src/EA.Weee.Core.Tests.Unit/AatfReturn/AatfSchemeDataTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace EA.Weee.Core.Tests.Unit.AatfReturn
+{
+    using EA.Weee.Core.AatfReturn;
+    using FakeItEasy;
+    using FluentAssertions;
+    using System;
+    using Xunit;
+
+    public class AatfSchemeDataTests
+    {
+        [Fact]
+        public void Construct_GivenNullSchemeAndObligatedReceivedData_ArgumentNullExceptionExpected()
```

Comment:
Split this test into 2 tests, 1 to test the scheme and the other to test the obligated received data. Just to make sure that both are working as expected